src/inc/AppxPackaging.hpp

```diff
@@ -645,6 +649,19 @@ enum tagLOCKTYPE
     };
 #endif 	/* __IAppxManifestReader3_INTERFACE_DEFINED__ */
 
+#ifndef __IAppxManifestReader4_INTERFACE_DEFINED__
+#define __IAppxManifestReader4_INTERFACE_DEFINED__
+
+    // {4579BB7C-741D-4161-B5A1-47BD3B78AD9B}
+    MSIX_INTERFACE(IAppxManifestReader4, 0xc43825ab, 0x69b7, 0x400a, 0x97, 0x09, 0xcc, 0x37, 0xf5, 0xa7, 0x2d, 0x24); // what are these numbers for?

```

Comment:
>// what are these numbers for? [](start = 118, length = 30)

The GUID of the interface.  Yours do not match the comment, please find the right one and update.